README.md

```diff
@@ -1,38 +1,66 @@
+	<a href="" target="__blank"><img alt="GitHub downloads" src="https://img.shields.io/github/downloads/cadriel/fluidd/total?style=flat-square"></a>
+	<a href="" target="__blank"><img alt="Docker Pulls" src="https://img.shields.io/docker/pulls/cadriel/fluidd?style=flat-square"></a>
+  	<a href="" target="__blank"><img alt="GitHub language count" src="https://img.shields.io/github/languages/count/cadriel/fluidd?style=flat-square"></a>
+	<img alt="GitHub commit activity" src="https://img.shields.io/github/commit-activity/m/cadriel/fluidd?style=flat-square"></a>
+</h4>
 
-## Support & Documentation
-See our [Docs](https://docs.fluidd.xyz).
-Join our [Discord!](https://discord.gg/GZ3D5tqfcF).
+<img align="right" src=".github\images\community-logos.png" alt="Community Logos" width="250"/>
 
-## Where to download?
-You can download the latest release [here](https://github.com/cadriel/fluidd/releases/latest).
+## Key Features
 
-Older releases can be found [here](https://github.com/cadriel/fluidd/releases).
+- Responsive UI made for *desktop, tablet & mobile*!
+- Customize you dashboard! Choose what you want to see and move every panel where YOU want! 
+- Built-in color and community themes, pick a logo you like!
+- Manage multiple printers from one Fluidd install!
+- Fast development cycle and active devs!
+- Multi language support!
+- [See our docs for a lot more!](https://docs.fluidd.xyz)
 
-## Docker
-We have an official docker image. This is updated for each release.
-https://hub.docker.com/r/cadriel/fluidd
+# Support & Documentation
+<p>
+	<a href="" target="__blank"><img alt="GitHub closed issues" src="https://img.shields.io/github/issues-closed/cadriel/fluidd?style=flat-square"></a>
+	<a href="" target="__blank"><img alt="Discord" src="https://img.shields.io/discord/762863525818662974?label=discord&style=flat-square&logo=discord"></a>
+</p>
+
+Looking for support? [Check out our docs](https://docs.fluidd.xyz) or [open an issue](https://github.com/cadriel/fluidd/issues/new/choose)!
+
+Have a killer idea you'd like to see implemented? Join our [Discord](https://discord.gg/GZ3D5tqfcF) to discuss bugs, features and share fireside fluidd stories.
+
+# Where can I get it?
+You can get your hands on the latest release of *fluidd* [here](https://github.com/cadriel/fluidd/releases/latest), while older releases can be found [here](https://github.com/cadriel/fluidd/releases).
+
+We also have an official docker image! This is updated for each release and can be found [here](https://hub.docker.com/r/cadriel/fluidd).
+
+
+# How do I use it?
 
-## How to use?
 Fluidd is distributed as a RaspbiOS image, with Fluidd pre-installed and setup alongside its dependencies.
 
 The recommended approach is to flash a fresh image using FluiddPI.
 
 Please see the [docs](https://docs.fluidd.xyz) for help with installation and configuration.
 
-### Credit
+# Credit
 A big thank you to;
+
 - the [Voron Community](http://vorondesign.com/)
```

Comment:
Should we add the other communities here?

Reply:
Great idea, let me add those!